Server/Program.cs

```diff
@@ -67,13 +67,6 @@ async Task LoadShines()
     c.Metadata["scenario"] = (byte?) 0;
     c.Metadata["2d"] = false;
     c.Metadata["speedrun"] = false;
-    foreach (Client client in server.ClientsConnected) {
```

Comment:
I guess since the reconnecting system is jank anyways it doesn't matter too much?

Reply:
Yes.

I moved sending the `lastGamePacket` to the `ResendPackets` method together with the other messages that need resending (that were not resend before).

The packet is still send like before, just a bit later now after the call to the `ClientJoined` handler. (In the `Server.cs` Line 292).

Because: The `ClientJoined` handler is only called on the "first" connection, and not on recurring connections. Now `ResendPackets` is triggered for connections for every new socket regardless if it's an existing or a new player.